docs/source/performance/perf-overview.md

```
@@ -21,49 +23,61 @@ The performance numbers below were collected using the steps described in this d
+### Hardware
+The following GPU variants were used for testing:
+- H100 SXM 80GB (DGX H100)
+- H200 SXM 141GB (DGX H200)
+- GH200 96GB HBM3 (480GB LPDDR5X)
+- B200 180GB (DGX B200)
+- GB200 192GB (GB200 NVL72)
+
+Other hardware variants may have different TDP, memory bandwidth, core count, or other features leading to performance differences on these workloads.
+
+### FP4 Models
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)